packages/node/src/indexer/api.service.ts

```diff
@@ -211,6 +211,27 @@ export class CosmosClient extends CosmWasmClient {
       throw e;
     }
   }
+
+  static handleError(e: Error): Error {
+    const formatted_error: Error = e;
+    try {
+      const message = JSON.parse(e.message);
+      if (
+        message.data &&
+        message.data.includes(`is not available, lowest height is`)
+      ) {
+        logger.info('here');
+        formatted_error.message = `${message.data}\nuse an archive/full node instead of a pruned node`;
```

Comment:
Do you have an example of when this is printed? It looks like it might easily be missed